docs/index.md

```diff
@@ -1596,16 +1593,16 @@ tests as shown below:
 <html>
   <head>
```

Comment:
<meta charset="utf-8" />

Reply:
I'm not sure what you're reviewing here, but it's not this PR...